multiview/embed/utils.py

```diff
@@ -0,0 +1,173 @@
+from scipy.stats import norm
+import numpy as np
+from scipy.sparse.linalg import svds
+
+
```

Comment:
fine to copy this stuff, just add something saying where you got it?

Reply:
See #46